cluster/src/main/java/com/hartwig/pipeline/execution/vm/InputDownload.java

```diff
@@ -25,7 +25,7 @@ private static String localPath(final GoogleStorageLocation sourceLocation) {
 
     @Override
     public String asBash() {
-        return format("gsutil -qm cp -n gs://%s/%s%s %s%s",
+        return format("gsutil -qm cp -r -n gs://%s/%s%s %s%s",
```

Comment:
We actually probably want to do this instead of what is currently on line 31, but wrapped in the same tertiary.